Networking/Networking/Model/OrderItem.swift

```diff
@@ -87,17 +92,18 @@ private extension OrderItem {
 
     enum CodingKeys: String, CodingKey {
         case itemID         = "id"
-        case name           = "name"
+        case name
         case productID      = "product_id"
-        case quantity       = "quantity"
-        case price          = "price"
-        case sku            = "sku"
-        case subtotal       = "subtotal"
+        case variationID    = "variation_id"
```

Comment:
I know this is completely off field, and it's just a question, but have we ever considered setting the .keyEncodingStrategy property of the JSONEncoder to .convertToSnakeCase?

If I recall correctly, that would make the encoder automatically convert coding keys between camel case and snake case (i.e. productId to product_id)

Reply:
Heck ya! I'll do that.

Reply:
Note that `OrderItem` is decodable-only. But I will do that for the models we have that are encodable!

Reply:
Follow up. I went to apply it to Refunds and I realized why I didn't do that in the first place. I changed the names of some of the fields in an attempt to add clarity to what they were. So converting to/from snake case isn't the main issue in this PR. It's converting the field names to completely different values. i.e. `case isAutomatedRefund = refunded_payment`

Reply:
Oh, I see. 🤷‍♂